General_gNMI/Get.resource

```diff
@@ -41,3 +41,10 @@ Iterate non-empty get path "${path}"
     When Verify Get of  ${path}
     Then Check last updates not empty
     [Teardown]    Teardown gNMI state
+
+Try getting sanity path
+    [Documentation]    Try reading sanity data from path declared by YAML variable "get_sanity_path".
+    ...                Fails the test if variable is not set.
+    Should Be String    ${get_sanity_path}  Missing "get_sanity_path" variable to get some device data.
```

Comment:
This does not work - if the variable is missing, robot bails out with its own error message. Looks like `Get Variable Value` builtin can be used for checking if a variable has been set; I'm not sure if that is needed, maybe the default error message is enough.

Reply:
right, i left original error for missing variable as "self-explanatory" (`Variable '${get_sanity_path}' not found.`), 
and added this as extra check...

Reply:
If the variable is indeed undefined, you get the built-in "not found" message, so this statement does not have any effect (except for triggering the error); if the variable is defined but not as a string, you get your "Missing variable" message, so the statement does have an effect, but with a misleading message.

Reply:
fixed the incorrect  error message in latest codebase